app/js/components/manageApps/ManageApps.jsx

```diff
@@ -182,7 +186,7 @@ export default class ManageApps extends React.Component {
                 });
                 return result;
               }).catch(error => {
-                const installedAddons = installedOwas.concat(installedModules);
```

Comment:
Is this change part of the ticket?

Reply:
The proper implementation had been removed in a previous PR. I was just putting it back.